jgrapht-ext/src/main/java/org/jgrapht/ext/DOTUtils.java

```diff
@@ -22,25 +22,30 @@
 import org.jgrapht.*;
 
 /**
+ * Class with DOT format related utilities.
+ * 
  * @author Christoph Zauner
+ * 
+ * @deprecated in favor of implementing the current functionality directly by the user
  */
+@Deprecated
 public class DOTUtils
```

Comment:
Another idea is to keep all this variables/method in the `DOTExporter` as public, but I think that they should live in their own utils class...